src/main/java/net/fabricmc/loom/configuration/fabricapi/FabricApiTesting.java

```diff
@@ -94,10 +99,23 @@ void configureTests(Action<GameTestSettings> action) {
 		}
 
 		if (settings.getEnableClientGameTests().get()) {
+			// Not ideal as there may be multiple resources directories, if this isnt correct the mod will need to override this.
```

Comment:
Hmm I think this is more likely to be wrong than right. I imagine most people will put their client gametest as a separate mod in a separate source set.

Reply:
Fixed, thanks it now uses the generated sourceset.